installing/installing_rhv/installing-rhv.adoc

```diff
@@ -0,0 +1,13 @@
+[id="installing-rhv"]
+= Installing a cluster on Red Hat Virtualization
+include::modules/common-attributes.adoc[]
+:context: installing-rhv
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster
+in a Red Hat Virtualization (RHV) environment.
+
+The installer that deploys OCP on RHV automates the process using what’s called “installer-provisioned infrastructure.” This installer is available for Linux and macOS only.
+
+This link:https://access.redhat.com/articles/4903411[Quickstart Guide: Installing OpenShift on Red Hat Virtualization] shows you how to get up and running with OCP on RHV.
```

Comment:
_To learn more about installing a {product-title} cluster on RHV, read link:https://access.redhat.com/articles/4903411[Quickstart Guide: Installing OpenShift on Red Hat Virtualization]_

Reply:
In the link text, I replaced "OpenShift" with {product-title}, okay?